artifactory.py

```diff
@@ -2585,15 +2593,15 @@ def get_build_info(self, build_name, build_number):
         return self._get_info(build_name, build_number)
 
     def _get_info(self, build_name, build_number=""):
-        url = build_name
+        url = requests.utils.quote(build_name, safe="")
```

Comment:
```suggestion
        # If a build name contains slash "/" it must be encoded, otherwise the part after the slash will be treated as a build number
        # maven-demo/1-build-snapshot => maven-demo%2F1-build-snapshot
        url = requests.utils.quote(build_name, safe="")
```

Reply:
Let's add a short comment about the case